Dockerfile

```diff
@@ -12,10 +12,32 @@ RUN curl -sL https://deb.nodesource.com/setup_6.x | bash \
     && apt-get install --no-install-recommends -y nodejs \
     && rm -rf /var/lib/apt/lists/*
 
+# Kosmtik with plugins
 RUN npm install -g kosmtik
 
+WORKDIR /usr/lib/node_modules/kosmtik/
```

Comment:
This is also not the perfect solution too. When the kosmtik-place-search plugin will be working, I would like to add it to repo and then what? If the user changed the file, we should parse it and add this plugin name in the proper place (maybe other than just the end of the file). It also means that every change a user makes to the config file will probably alarm git and make updating code harder. It's not a good idea to treat configuration like the code.

That's why I've tried a brute force approach - at least configuration file will be in sync with installed plugins. So I guess I will stick to that way, but I will just use the test - if the config file is already here, user is on his own, since we don't know what it looks like. OK, we can even test if this is original file we left here before (with MD5 hash), so we can update it automatically, but this might be overengineering.